app/main.py

```
@@ -1,6 +1,31 @@
+import os
+import requests
+
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+
+    if not api_key:
+        print("API_KEY environment variable not set.")
+        return
```

it's better raise error instead of simple return